community/mailinglists.md

```diff
@@ -0,0 +1,47 @@
+# Mailing Lists
+
+Here are the [mailing lists](https://mta.openssl.org/) we run. You must
+be a member of the list to post to it. Anything you post to a list,
+including the E-mail address you posted from, will be sent to and seen
+by all other members of the list.
+
+  ------------------------------------------------------------------------------- -----------------------------------------------------------------------------------
```

Comment:
This table doesn't render at all well.  It needs conversion to a MD table.

Reply:
Reformated